reversed_func.py

```diff
@@ -0,0 +1,21 @@
+def reversed_func(text: str) -> str:
```

Comment:
Disagree with the name of the function. By forgetting that I know what it does and trying to derive that information from the function's name, I would guess that reversed_func does something to the function so it becomes reversed. Try something different, something more self-speaking